### ISSUE TITLE
Cathedral of Eternal Night: Fel Beast rewards give bad data

### ISSUE THREAD

**Issue by Arcanemagus** [DrOxlotus/LastSeen#2]
This _may_ be a general issue with all dungeon WQ rewards, however I have only personally seen it with [Cathedral of Eternal Night: Fel Beast](https://www.wowhead.com/quest=46867).

Blood Elf Paladin:
![image](https://user-images.githubusercontent.com/427137/59585813-2f071c00-9096-11e9-9113-bb6bd8cf8bcc.png)

Item in the LastSeen database:
```lua
    [144502] = {
        ["itemRarity"] = 4,
        ["itemName"] = "Shadowed Grips of the Nathrezim",
        ["itemType"] = "Armor",
        ["key"] = "144502708310497114971151169711011097366107939FEC49",
        ["source"] = "Cathedral of Eternal Night: Fel Beast",
        ["itemLink"] = "|cff0070dd|Hitem:144502::::::::120:70::::::|h[Shadowed Grips of the Nathrezim]|h|r",
        ["lootDate"] = "06/16/19",
    },
```

Night Elf Druid:
![image](https://user-images.githubusercontent.com/427137/59585832-3a5a4780-9096-11e9-9c54-db91532906a3.png)

And in the LastSeen DB:
```lua
	[144487] = {
		["itemRarity"] = 4,
		["itemName"] = "Treads of Creeping Shadows",
		["itemType"] = "Armor",
		["key"] = "144487708310497114971151169711011097366107939FEC49",
		["source"] = "Cathedral of Eternal Night: Fel Beast",
		["itemLink"] = "|cff0070dd|Hitem:144487::::::::120:103::::::|h[Treads of Creeping Shadows]|h|r",
		["lootDate"] = "06/16/19",
	},
```

Version: https://github.com/DrOxlotus/LastSeen/commit/9c22a38507d40476217c58a1f7fffbfd7c2f992e

**Comment by DrOxlotus** [comment]
What an interesting dilemma. I'll take a look tomorrow.

**Comment by DrOxlotus** [comment]
I had this problem appear last night in Tol Dagor for a pair of epic bracers. I didn't capture which field was nil, but it would likely need to be location. I'll see what I can do.

**Comment by DrOxlotus** [comment]
Should be fixed in 666d608.
